src/Features/Core/Portable/Diagnostics/DiagnosticResultSerializer.cs

```diff
@@ -15,8 +15,10 @@ namespace Microsoft.CodeAnalysis.Diagnostics
 {
     internal static class DiagnosticResultSerializer
     {
-        public static void Serialize(ObjectWriter writer, DiagnosticAnalysisResultMap<string, DiagnosticAnalysisResultBuilder> result, CancellationToken cancellationToken)
+        public static (int diagnostics, int telemetry, int exceptions) Serialize(
```

Comment:
💭 I'm not sure I understand why these are changing.

Reply:
to log that information. I believe this data is logged now? not strictly related to 64bit work. but was one of data I was using when looking at 64bit perf.